pliers/transformers/base.py

```diff
@@ -215,7 +216,7 @@ def __init__(self, batch_size=None, *args, **kwargs):
             self._batch_size = batch_size
         super(BatchTransformerMixin, self).__init__(*args, **kwargs)
 
-    def _iterate(self, stims, *args, **kwargs):
+    def _iterate(self, stims, validation='strict', *args, **kwargs):
```

Comment:
I'm guessing this is just a hook left for later implementation? That's fine with me, just making sure it's not meant to be doing anything right now.

Reply:
That is there to unpack `validation` from `**kwargs`, such that `_transform` isn't called with an unexpected argument. That being said, I'm actually not sure why we have `*args` and `**kwargs` passed into `_transform` since every call should only accept one argument for the stim.